kafka/metrics.go

```diff
@@ -73,43 +101,168 @@ func newKgoHooks(mp metric.MeterProvider, namespace, topicPrefix string,
 	topicAttributeFunc TopicAttributeFunc,
 ) (*metricHooks, error) {
 	m := mp.Meter(instrumentName)
+
+	// kotel metrics
+
+	// connects and disconnects
+	connects, err := m.Int64Counter(
+		"messaging.kafka.connects.count",
```

Comment:
Q. Should we define constant values for the kotel metrics, too? On the other hands, it also make sense to inline them to distinguish from our custom metric.

Reply:
Makes sense. thanks for the clarification 👍